container-toolkit/install-guide.md

```diff
@@ -10,16 +10,22 @@
 
 ### Prerequisites
 
-Install the NVIDIA GPU driver for your Linux distribution.
-NVIDIA recommends installing the driver by using the package manager for your distribution.
+1. Read [this section](./supported-platforms.md) about platform support.
 
+2. Install the NVIDIA GPU driver for your Linux distribution.
+NVIDIA recommends installing the driver by using the package manager for your distribution.
 For information about installing the driver with a package manager, refer to
 the [_NVIDIA Driver Installation Quickstart Guide_](https://docs.nvidia.com/datacenter/tesla/tesla-installation-notes/index.html).
+Alternatively, you can install the driver by [downloading](https://www.nvidia.com/en-us/drivers/) a `.run` installer.
 
-Alternatively, you can install the driver by downloading a `.run` installer.
-Refer to the NVIDIA [Official Drivers](https://www.nvidia.com/Download/index.aspx?lang=en-us) page.
```

Comment:
Is there a reason why we got ride of this link? I dont remember if this has come up before in a previous comments. thanks

Reply:
No removal. I replaced it with a more modern URL. You will notice in your browser that the old URL redirects to `/en-us/drivers`.

Personally, I like to look at changes like this in the rendered view.

We should also work towards using often called "semantic line breaks" to keep the markdown source diff better reviewable.